ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-786] Do not display outgoings payments if not asked

## Description of change

Outgoings payment was being displayed when an application was returned, despite the outgoings section not being answered during the initial application

The logic for outgoings payments, income payments and income benefits has been amended to use attributes that have been added since the section was added `has_no_other_outgoings`, `has_no_income_benefits` and `has_no_income_payments`

The logic for the task list is also amended to correctly show the not applicable tag for the outgoings an capital sections 

## Link to relevant ticket
[CRIMAPP-786](https://dsdmoj.atlassian.net/browse/CRIMAPP-786)

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

<img width="950" alt="Screenshot 2024-04-29 at 09 02 54" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/63cc9ce0-3c38-447d-b92b-beacd7051215">

### After changes:
<img width="950" alt="Screenshot 2024-04-29 at 08 59 49" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/5aec5544-09f3-4186-a85c-32f6f2bb4c9f">


## How to manually test the feature

Create an application that is:

- Summary only
- Has nino 
- No passporting benefit 
- Income under £12475
- Answered No to the freezing order question
- Answered No to all income questions

Submit the application
Return the application in review
View the application in apply in the returned application tab 
The `payments your client pays` section should not be visible

[CRIMAPP-786]: https://dsdmoj.atlassian.net/browse/CRIMAPP-786?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
